tests/test1010.py

```diff
@@ -0,0 +1,54 @@
+# test some libuninameslist functions (if the library is included).
+
+import sys, fontforge
+
+cnt = UnicodeBlockCountFromLib()
+if ( cnt < 1 ):
+  print("No NamesList library - Nothing to test.")
+  return
+
+if ( cnt < 294 ):
+  print("Pre-20170701 or older library - Nothing to test beyond here.")
+  return
+
+ver = fontforge.UnicodeNamesListVersion()
+Print("Libuninameslist version is :",ver)
+
+# test Names2 functions (if we have libuninameslist version>=10.9)
+if ( cnt < 305 ):
+  print("Pre-20180408 or older library - Nothing to test beyond here.")
+  return
+
+cnt = fontforge.UnicodeNames2GetCntFromLib()
+print("Libuninameslist internal table has ",cnt," Names2.")
+if ( cnt < 25 ):
+  print("Test Names2 functions. Must be libuninameslist 10.9 or newer.")
+  raise ValueError("Expected 25 or more values that have Names2.")
+
+# none of these are -1 if library loaded
+u0 = fontforge.UnicodeNames2NxtUniFromLib(0)
+u1 = fontforge.UnicodeNames2NxtUniFromLib(1)
+u2 = fontforge.UnicodeNames2NxtUniFromLib(2)
+print("Sample of internal table has ",u0,", ",u1,", ",u2," as the first 3 unicode values.")
+if ( u0 < 0 || u1 < 1 || u2 < 2 ):
```

Comment:
`or`, not `||`. Did you run this locally?

Reply:
Still need to get python dev loaded. This can get removed in a squash-merge.